### PR TITLE
Campaign preprocess configs olé 5

### PR DESCRIPTION
**Pull request by weerd**
Refs #5147
#### What's this PR do?

This PR updates the configuration variables for User section of variables within **DoSomething Config -> DoSomething Campaign**.
#### Where should the reviewer start?

Just confirming that the code looks correct.
#### Any background context you want to provide?

The fields are translated both on the main site page, user-facing output and within the input fields in the CMS. They have all been manually tested to confirm that if a translation is available, the translation is used.
#### What are the relevant tickets?
#5147

---

@angaither

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/includes/auth/login.inc [hunk 1]
```diff
@@ -128,12 +128,11 @@ function paraneue_dosomething_pass_submit($form, &$form_state) {
...
 function paraneue_dosomething_pass_set_message() {
-  $default_msg = t(
-    'If your email address exists in our database, you will receive ' .
-    'a password recovery link at your email address in a few minutes.'
-  );
+  $default_msg = 'If your email address exists in our database, you will receive a password recovery link at your email address in a few minutes.';
   $msg = variable_get('dosomething_user_reset_password_copy', $default_msg);
-  drupal_set_message($msg);
+
+  // @TODO: #Global; translation substitution doesn't seem to work correctly with this. Not sure why...?
```
Comment (angaither) [inline lib/themes/dosomething/paraneue_dosomething/includes/auth/login.inc:134]: interesting... for all drupal set messages?

**Comment by angaither** [comment]
:+1:

File: lib/themes/dosomething/paraneue_dosomething/includes/auth/login.inc [hunk 2]
```diff
@@ -128,12 +128,11 @@ function paraneue_dosomething_pass_submit($form, &$form_state) {
...
+  drupal_set_message(t($msg));
```
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/includes/auth/login.inc:135]: Made a note so I can move on, but was having issues seeing the translated copy show up in the Drupal message. Not sure what is up, but this _should_ be the proper way to output translated copy when using `drupal_set_message()`... Will need to come back to it. :grimacing:

Comment (angaither) [inline lib/themes/dosomething/paraneue_dosomething/includes/auth/login.inc:135]: References #5142

File: lib/themes/dosomething/paraneue_dosomething/includes/auth/login.inc (hunk 1 above)
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/includes/auth/login.inc:134]: Not sure. This is the only one I've had to test so far with the config stuff, but something I'm noting and will need to figure out. Based on what I've found and read, it _should_ be working...

**Pull request merged**